static/js/vue-cdr-access/src/components/browseSearch.vue

```diff
@@ -78,16 +78,27 @@ Search form component displayed on full record pages, allowing keyword searches
 
         methods: {
             getResults() {
-                let update_params = { anywhere: encodeURIComponent(this.search_query) };
-                this.routeWithParams(this.urlParams(update_params, 'displayRecords'));
+                const encode_search = { anywhere: encodeURIComponent(this.search_query), start: 0 };
+                const update_parameters = this.urlParams(encode_search, 'displayRecords');
+                this.routeWithParams(update_parameters);
             },
 
             clearSearch() {
-                this.routeWithParams(this.removeQueryParameters(this.allPossibleSearchParameters));
+                this.clear(this.allPossibleSearchParameters);
             },
 
             clearAllFacets() {
-                this.routeWithParams(this.removeQueryParameters(this.possibleFacetFields));
+                this.clear(this.possibleFacetFields);
+            },
+
+            /**
+             * Clears facets/searches
+             * @param param_to_clear
+             */
+            clear(param_to_clear) {
```

Comment:
maybe `clearParameters` to differentiate it from other possible clearing operations (like clearing the search box or something)